incidents/transform.py

```diff
@@ -0,0 +1,89 @@
+"""Functions to transform and clean extracted data"""
```

Comment:
Can add a space after the module docstring for styling purposes (and consistency with the other scripts)